haaska.py

```diff
@@ -124,7 +124,11 @@ def entity_domain(x):
         return x['entity_id'].split('.', 1)[0]
 
     def is_supported_entity(x):
-        return entity_domain(x) in ['light', 'switch', 'group', 'scene', 'media_player', 'input_boolean', 'script']
+        allowed_entities = ['group', 'input_boolean', 'light', 'media_player', 'scene', 'script', 'switch']
+        cfg = get_config()
```

Comment:
Hm, I think it might be better to just make "cfg" global, instead of reopening and reparsing for every invocation of is_supported_entity